components/04-views/login/login--without-banner.hbs

```diff
@@ -0,0 +1,86 @@
+{{ render '@header' }}
+
+<main class="container login">
+
+    <h1 class="login__title">
+        {{ title }}
+    </h1>
+
+    <div class="login__content">
+
+        <div class="login__tabs">
+
+            <div class="
+                     tab {{#if tabs.modifier}}tab--{{tabs.modifier}}{{/if}}
```

Comment:
Wouldn't it be better to just use a `class` variable and put a modifier inside ?

Reply:
sure. anyway it was some old code as it wasn't used